Gemfile

```diff
@@ -201,7 +201,7 @@ gem 'rubyzip', require: 'zip'
 gem 'nokogiri', '>= 1.6.8'
 
 # Polyglot support
-gem 'coursemology-polyglot', '>= 0.1.0'
+gem 'coursemology-polyglot', github: 'Coursemology/polyglot'
```

Comment:
Yes, it's needed here to populate the db table. Hmm I don't think it's necessary to specify the github repo though. Should the gem version be updated?